client/go/auth0/auth0.go

```diff
@@ -166,13 +166,7 @@ func (a *Auth0) PrepareSystem(ctx context.Context) (*System, error) {
 
 		res, err := tr.Refresh(ctx, a.system)
 		if err != nil {
-			// ask and guide the user through the login process:
```

Comment:
Not entirely sure about this one. Automatically renewing the token makes it all a bit easier for the user. But then again, if there is an intermittent network issue that casued the refresh to fail, starting the entire flow again is not really needed.

What is the lifetime of a refresh token?

Reply:
Before my change this didn't work as the re-authentication output gets mangled with the spinner output. It's better to return the error and tell the user to re-authenticate in this case.

Note that this is different from refresh. Here you're doing a full re-authentication with the browser flow if refresh failed. Re-running `vespa auth login` is a very minor inconvenience in this case.

Reply:
ok for me